save-api/build.gradle.kts

```diff
@@ -26,13 +23,24 @@ kotlin {
 dependencies {
     implementation(projects.saveCloudCommon)
     implementation(libs.save.common.jvm)
-    implementation(libs.kotlinx.cli)
     implementation(libs.log4j)
     implementation(libs.log4j.slf4j.impl)
     implementation(libs.ktor.client.apache)
     implementation(libs.ktor.client.auth)
     implementation(libs.ktor.client.core)
     implementation(libs.ktor.client.serialization)
     implementation(libs.ktor.client.logging)
-    implementation(libs.kotlinx.serialization.properties)
 }
+
+publishing {
+    publications {
+        create<MavenPublication>("maven") {
+            groupId = "org.cqfn"
+            artifactId = "save-cloud-api"
+            version = "0.1.2"
```

Comment:
Is `project.version` fine here?

Reply:
Actually I'm lost in our versioning in save-cloud

Reply:
Changed to `v1`

Reply:
Let's still have `project.version` and have a common versioning for the whole repository. Once we add breaking changes into API, that require a new release of the client library, we will create a git tag on the whole repository and release the library under the corresponding version.